common/hooks/pre-pkg/99-pkglint-subpkgs.sh

```diff
@@ -18,21 +18,22 @@ hook() {
 
     subpkgs=$(get_subpkgs)
 
-    subpackages="${subpackages// /$'\n'}"
-
     # Sort the strings so they can be compare for equality
-    subpkgs="$(printf "%s\\n" "$subpkgs" | sort)"
-    subpackages="$(printf "%s\\n" "$subpackages" | sort)"
+    subpkgs="$(printf "%s\\n" $subpkgs | sort)"
+    subpackages="$(printf "%s\\n" $subpackages | sort)"
 
     if [ "$subpackages" = "$subpkgs" ]; then
         return 0
     fi
 
     # XXX: Make the sed call work when subpackages has multiple lines
```

Comment:
isn't that comment obsolete with your fix?

Reply:
Ah, yes, I'll remove it, too.